spec.yml

```diff
@@ -3265,6 +3362,140 @@ paths:
         - OAuth2: []
       tags:
         - pmi
+  /property-entries:
```

Comment:
@danschultz , @MadisonEhlers-Vertex , I assume it's OK for this property-related stuff to go out.